spec/policies/resource_policy_spec.rb

```diff
@@ -9,11 +9,36 @@
     let(:user) { nil }
 
     it { is_expected.to permit_action(:show) }
+
+    it { is_expected.to forbid_action(:index)   }
+    it { is_expected.to forbid_action(:new)     }
+    it { is_expected.to forbid_action(:create)  }
+    it { is_expected.to forbid_action(:update)  }
+    it { is_expected.to forbid_action(:edit)    }
+    it { is_expected.to forbid_action(:destroy) }
   end
 
   context 'when user' do
     let(:user) { FactoryGirl.create(:user) }
 
     it { is_expected.to permit_action(:show) }
+    it { is_expected.to permit_action(:index)   }
+    it { is_expected.to permit_action(:new)     }
+    it { is_expected.to permit_action(:create)  }
+    it { is_expected.to forbid_action(:update)  }
```

Comment:
D'oh. I completely misread the text. I didn't read the `forbid_` part of the `forbid_action`. Please disregard my comment :)